A6.py

```diff
@@ -0,0 +1,60 @@
+"""
+Write a function that when given a tic-tac-toe board (two 
+dimensional array of strings that are either 'X', 'O', or ''), 
+return the winner of the board. If there is no winner, return None.
+
+Here is the pull request.
+
+"""
+
+from operator import add as list_concat
+
+def get_winner(board):
+	"""
+	Returns None if no winner. Otherwise, returns "X" or "O" as winner.
+	"""
+
+	winning_lines = get_winning_lines(board)
+	for winning_line in winning_lines:
```

Comment:
I don't understand what is happening. This is an overly confusing way to just check if a line is won